tutorials/vue/vue-dropdown.md

```diff
@@ -0,0 +1,69 @@
+Using Bootstrap, Vue, and Popper.js, you can make a more modern dropdown as
+opposed to the `<select>` tag in HTML.
+
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous"/>
```

Comment:
I'd recommend creating a separate `.html` file in the `examples/vue` directory and link out to that page, rather than include bootstrap here. Importing bootstrap has the unfortunate side effect of breaking the rest of the layout:

![image](https://user-images.githubusercontent.com/1620265/111554500-e1cd4800-875c-11eb-8270-4d7c9a0ae24c.png)